actions/templates.go

```diff
@@ -205,17 +213,17 @@ func PutTemplate(c *gin.Context) {
 		return
 	}
 
-	template2, err := storage.GetTemplateByName(c, body.Name, u.ID)
-	if err == nil && template.ID != template2.ID {
+	_, err = storage.GetTemplateByName(c, body.Name, u.ID)
+	if err == nil {
```

Comment:
the template.ID != template2.ID should stay, if it's the case of the same template, the name **belongs** to that template so we should not throw an error here.